src/lsp/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2022 IWANABETHATGUY
```

Comment:
What's up with this? Is this derived from an upstream MIT-licensed project?

Reply:
I initially used this part from [the template](https://github.com/ebkalderon/tower-lsp/tree/master#ecosystem) in the tower-lsp repo, but after learning and modifications, I ended up removing almost everything from it. Now, it closely resembles [the `tower-lsp` examples](https://github.com/ebkalderon/tower-lsp/blob/master/examples/stdio.rs) rather than the original template.